Src/MoneyFox.Shared/Manager/EndOfMonthManager.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using MoneyFox.Shared.Interfaces;
+using MoneyFox.Shared.Interfaces.Repositories;
+using MoneyFox.Shared.Model;
+using MoneyFox.Shared.Resources;
+using MvvmCross.Platform;
+using MvvmCross.Platform.Platform;
+using System.Collections.ObjectModel;
+using MoneyFox.Shared.Repositories;
+
+
+namespace MoneyFox.Shared.Manager
+{
+    public class EndOfMonthManager : IEndOfMonthManager
+    {
+        private readonly IPaymentRepository paymentRepository;
+        private readonly IAccountRepository accountRepository;
+        
+        public EndOfMonthManager( IPaymentRepository paymentRepository, IAccountRepository accountRepository)
+        {
+            this.paymentRepository = paymentRepository;
+            this.accountRepository = accountRepository;
+            
+        }
+       
+
+        public void AssignToAccounts()
+        {
+            foreach(Account x in accountRepository.GetList())
+            {
+                DeterminEndThroughAccounts(x);
+            }
+        }
+
+
+        /*
```

Comment:
make the comments with /// this generates a documentation header (for example in the data access classes are some of them).